src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/security/CcdEventAuthorizor.java

```diff
@@ -1,15 +1,20 @@
 package uk.gov.hmcts.reform.iacaseapi.infrastructure.security;
 
+import static org.slf4j.LoggerFactory.getLogger;
+
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
+
 import org.springframework.security.access.AccessDeniedException;
 import uk.gov.hmcts.reform.iacaseapi.domain.UserDetailsProvider;
 import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
 
 public class CcdEventAuthorizor {
 
+    private static final org.slf4j.Logger log = getLogger(CcdEventAuthorizor.class);
```

Comment:
Can use the @Slf4j annotation from Lombok which saves you from declaring the logger

Reply:
Ok, let me change all places in ia-case-api for being consistent (please see @jkumar-moj comment)